Source/WebKit/UIProcess/RemoteLayerTree/ios/ScrollingTreeScrollingNodeDelegateIOS.mm

```diff
@@ -265,6 +263,8 @@ - (void)cancelPointersForGestureRecognizer:(UIGestureRecognizer *)gestureRecogni
         }
         if (scrollingStateNode.hasChangedProperty(ScrollingStateNode::Property::ScrollableAreaParams)) {
             auto params = scrollingStateNode.scrollableAreaParameters();
+            scrollView.bouncesHorizontally = params.horizontalOverscrollBehavior != OverscrollBehavior::None;
```

Comment:
Shouldn't this be calling `_setBouncesInternal`?

Reply:
This code is only called for scrolling nodes, not the main document. Is it necessary to keep track of this state even if the client cannot set its value?

Reply:
Ah you're right. We need to refactor the code so that main scroller and subscroller updates share code.

Reply:
Good idea. Fixed.